libzeth/test/core/utils_test.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+#include "libzeth/core/utils.hpp"
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+static const size_t DUMMY_BUFFER_SIZE = 16;
+
+static const uint8_t dummy_bytes[DUMMY_BUFFER_SIZE] = {
```

Comment:
Remark: We need to be consistent with styling. Here there are 2 static consts but 2 naming styles (snake_case and UPPER_CASE).
We will deal with these in another PR once https://github.com/clearmatics/zeth/pull/208 is merged since I have started to configure all these things in the clang-tidy config file.
Spoiler: I have set the naming convention for static const to be UPPER_CASE

Reply:
I think there is scope for a difference between genuine "constants" (traditionally represented as `#define`s with all caps, which may completely disappear at compile time - e.g. the `const size_t` here) and structured data consisting of constant values.
As discussed, let's see what the linter tools flag / allow and make a decision.

Reply:
Yes, the tools provides an extensive set of possible configurations (here's a subset of such checks for identifier naming: https://clang.llvm.org/extra/clang-tidy/checks/readability-identifier-naming.html).
Let's refine this in a subsequent PR, and iterate on the coding conventions